.travis.yml

```diff
@@ -1,15 +1,52 @@
-branches:
-  only:
-    - master
-    - develop
+sudo: false
 language: node_js
 node_js:
   - "8"
-sudo: required
-install:
-  - yarn
-  - sudo pip install proselint
-before_script:
-  - source ./src/scripts/env.sh
-script:
-  - bash ./src/scripts/deploy.sh
+
+cache:
+  yarn: true
+  directories:
+    - node_modules
+    - dist
+    - .cache
+
+stages:
+  - Build
+  - Post-build
+
+jobs:
+  fast_finish: true
+  allow_failures:
+    - name: External Link Check
+
+  include:
+    - stage: Build
+      name: Lint and Build
+      before_install: npm install --global yarn
+      install: yarn
+      script:
+        - yarn lint:js
+        - yarn lint:markdown
+        - yarn lint:social
+        - yarn build
+        - yarn lint:links
```

Comment:
`yarn lint:links` is failing too https://travis-ci.org/webpack/webpack.js.org/jobs/411353517

Reply:
Random badge timeout. They seem to happen a lot. Might want to mark those as optional with a --todo